libraries/_helper.rb

```diff
@@ -173,7 +211,7 @@ def activate_splunk
       FileUtils.touch(splunk_lock_file)
     end
 
-    def is_splunk_enabled?
+    def splunk_enabled?
```

Comment:
That was part of the recomendations with Rubocop I think I can at least put it on the commit comment.. Thanks for the observation..